README.md

```diff
@@ -147,10 +147,50 @@ The derive function is expected to return an object with the desired target fiel
   - `fields` is an object containing each of the `from` fields. Each field will contain their current localized values (i.e. `fields == {myField: {'en-US': 'my field value'}}`)
   - `locale` one of the locales in the space being transformed
 
-   The return value must be an object with the same keys as specified in `derivedFields`. Their values will be written to the respective new entry fields for the current locale (i.e. `{nameField: 'myNewValue'}`)
+  The return value must be an object with the same keys as specified in `derivedFields`. Their values will be written to the respective new entry fields for the current locale (i.e. `{nameField: 'myNewValue'}`)
 
 - **`shouldPublish : boolean`** _(optional)_ – If `true`, both the source and the derived entries will be published. If `false`, both will remain in draft state (default `true`)
 
+### `config`
+
+There may be cases where you want to use Contentful API features that are not supported by the `migration` object. You have access to the internal configuration of the running migration.
+
+```javascript
+module.exports = function (migration, {makeRequest, spaceId, accessToken}) {
```

Comment:
Our code style demands spacing around objects braces, let's do the same in the readme.